content/6_Advanced/Segtree_Beats.mdx

```diff
@@ -1,21 +1,506 @@
 ---
 id: segtree-beats
 title: 'Segment Tree Beats'
-author: Benjamin Qi
+author: Benjamin Qi, Dustin Miao
 prerequisites:
   - RURQ
-description: '?'
+description: 'Perform chmin and chmax range updates'
 frequency: 1
 ---
 
-<IncompleteSection />
+# Segment Tree Beats
```

Comment:
Use an ex
```suggestion
## Segment Tree Beats
```

Reply:
for consistency, I don't think `#` alone is used anywhere. Increase depth of children as well.